index.html

```diff
@@ -49,5 +49,191 @@ <h1 id="name">
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="portfolio">
+            <div class="project">
+                <div class="work-img">
+                    <img src="/assets/images/projects/anoter-one.png" alt="tonic image">
+                </div>
+                <div class="work-description">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <ul class="canopy">
+                        <li>canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="about">
+        <div class="about-container">
+            <h1 class="name">About Myself</h1>
+            <p class="details">
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a href="#" class="connect">let's connect</a>
+            <ul class="socials">
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/twitter.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/Linkedin.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/microverse.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/github.svg" alt=""></i>
+                </a></li>
+                <li><a href="" >
+                    <i><img class="icons" src="/assets/images/icons/peace.svg" alt=""></i>
+                </a></li>
+            </ul>
+            <button class="btn resume">Get my resume</button>
+        </div>
+        <div class="skills">
+            <div class="languages">
```

Comment:
- [ ] To follow best practices and semantics, kindly consider taking a `ul` here instead of `<div class="skills">` and the all three `languages`, `frameworks` and `skills` would be `li`s. 
you dont need to change the rest of the code. the code you are have inside `<div class="languages">` is all fine.